### DISCUSSION TITLE
Inertial Drift

### DISCUSSION THREAD

**Discussion by By-M00NSpace** [skyline-emu/skyline#1505] [Game Requests]
I require this game
![71bAoJaPknL _SY445_](https://user-images.githubusercontent.com/83492639/172814023-f085d508-71a7-4910-b189-f415e72df31d.jpg)

**Comment by mateoalfaro** [comment]
It sure is on the Eshop tho

**Reply by Crytonics** [reply]
He asking to test in Skyline
